### PR TITLE
Convert tomcat 7 tests from groovy to java

### PR DESCRIPTION
**Pull request by jaydeluca**
Related to #7195

### CODE REVIEW

File: instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+import java.nio.file.Files;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.catalina.Context;
+import org.apache.catalina.LifecycleException;
+import org.apache.catalina.core.StandardHost;
+import org.apache.catalina.startup.Tomcat;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class TomcatHandlerTest extends AbstractHttpServerTest<Tomcat> {
```
Comment (steverao) [inline instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java:42]:
```suggestion
class TomcatHandlerTest extends AbstractHttpServerTest<Tomcat> {
```

File: instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+
+  @RegisterExtension
+  static final InstrumentationExtension testing = HttpServerInstrumentationExtension.forAgent();
+
+  private static final List<ServerEndpoint> serverEndpointsList =
+      Arrays.asList(
```
Comment (steverao) [inline instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java:48]: Use static import here?

Comment (jaydeluca) [reply instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java:48]: all of these endpoint names [are statically imported](https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/11402/files#diff-171264ab00659ef34034d4646a9b9656d89e794059955a8f4ba81ac860a0852aR8-R20), is there a different way you are suggesting?

Comment (trask) [reply instrumentation/tomcat/tomcat-7.0/javaagent/src/test/java/io/opentelemetry/javaagent/instrumentation/tomcat/v7_0/TomcatHandlerTest.java:48]: ah yes, that's typical in this repo: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/contributing/style-guideline.md#static-imports

**Review by laurit — APPROVED** [review]

**Pull request merged**